experimental/packages/otlp-transformer/src/trace/index.ts

```diff
@@ -20,16 +20,16 @@ import { toAttributes } from '../common/internal';
 import { sdkSpanToOtlpSpan } from './internal';
 import { IExportTraceServiceRequest } from './types';
 
-export function createExportTraceServiceRequest(spans: ReadableSpan[]): IExportTraceServiceRequest | null {
+export function createExportTraceServiceRequest(spans: ReadableSpan[], useHex?: boolean): IExportTraceServiceRequest {
   return {
     resourceSpans: spanRecordsToResourceSpans(spans).map(({ resource, resourceSpans, resourceSchemaUrl }) => ({
       resource: {
         attributes: toAttributes(resource.attributes),
         droppedAttributesCount: 0,
       },
-      instrumentationLibrarySpans: resourceSpans.map(({ instrumentationLibrary, instrumentationLibrarySpans, librarySchemaUrl }) => ({
-        instrumentationLibrary,
-        spans: instrumentationLibrarySpans.map(sdkSpanToOtlpSpan),
+      scopeSpans: resourceSpans.map(({ instrumentationLibrary, instrumentationLibrarySpans, librarySchemaUrl }) => ({
+        scope: instrumentationLibrary,
+        spans: instrumentationLibrarySpans.map(span => sdkSpanToOtlpSpan(span, useHex)),
```

Comment:
Since instrumentationLibrary is exposed to sdk user we can't change it in the sdk but we can in the otlp transformer which if i'm not mistaken has been done here ?

Reply:
We should use it in the rest of the packages as well, yes. We can still change it in the Metrics SDK, but it will be a bit more tricky for the Trace SDK as that could break user code and 1.0 is already out. I created an issue for that (#2937). Renaming `instrumentationLibrary` -> `instrumentationScope` was initially discussed in #2656.

I did not change it in the Metrics SDK as I feared it would blow up the diff of this PR too much. I could do it in a follow up, though :slightly_smiling_face:

Reply:
Changing the interface is one thing, but we have many variables in this file that are internal to the package and are still called `instrumentationLibrary`. Specifically in `spanRecordsToResourceSpans` and it's returned object.
I think this is a good chance to align at least what we can and is easy and keep the code consistent with naming where possible.
WDYT?